apps/arena/priv/config.json

```diff
@@ -80,6 +80,10 @@
         "shape": "polygon",
         "type": "dynamic",
         "vertices": [
+          {
+            "x": 3350.0,
+            "y": 4300.0
+          },
```

Comment:
Why is this moved here?

Reply:
Yeah this change isn't needed at all, restored